src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml

```diff
@@ -654,10 +660,15 @@
                                         AllowsTransparency="True"
                                         MouseLeave="OnPopupMouseLeave">
             <uicontrols:LibraryToolTipPopup.IsOpen>
-                <Binding ElementName="libraryToolTipPopup"
-                         Path="DataContext"
-                         Mode="OneWay"
-                         Converter="{StaticResource NodeSearchElementVMToBoolConverter}" />
+                <MultiBinding Converter="{StaticResource MultiBoolToBoolConverter}">
+                    <Binding Path="DataContext.ShowTooltipForTopResult"
+                             Mode="OneWay"
+                             RelativeSource="{RelativeSource FindAncestor, AncestorType={x:Type search:SearchView}}"/>
+                    <Binding ElementName="libraryToolTipPopup"
+                             Path="DataContext"
+                             Mode="OneWay"
+                             Converter="{StaticResource NodeSearchElementVMToBoolConverter}" />
+                </MultiBinding>
```

Comment:
We used binding, because we had to close tooltip with delay. If we use just IsOpen = true/false, then tooltip will open and close immediately. We didn't want to close it immediately, because there were link and code to copy. Both of these things are gone, and, maybe, we do not need to close it with delay. 
@elayabharath, what to you think? Should we close tooltip immediately?